Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM node:16-alpine
-
+FROM --platform=$BUILDPLATFORM node:16-alpine as builder
 WORKDIR /home/node/airbyte
-RUN npm install -g npm@7 tsc
+RUN npm install --location=global npm@7 tsc
```

Comment:
Should we use yarn instead of npm? It's installed by default in the node docker image. Don't care really.